### PR TITLE
Deployed with Github pages

### PR DESCRIPTION
**Pull request by edubew**
## In this milestone:
- I have Deployed my portfolio with Github pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,14 +10,35 @@
 - CSS
 - LINTERS
 
+## Live Demo
+[Winnie's Portfolio](https://edubew.github.io/winnie-s/portfolio/)
```
Comment (Deepakdanger) [inline README.md:14]: - [ ] The provided link is broken or not working, Kindly add the correct `deployed website link` in the `Readme` file.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @edubew, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by edubew** [comment]
Hello,
I am following all the steps provided in the documentation but my link
still won't work. I don't know how else to fix it

**Review by jenadiusnicholaus — COMMENTED** [review]
Hi @edubew 

Good job !
[Optional]

I think you are missing `Live link` part in you ReadMe file

**Review by AnselemOdims — APPROVED** [review]
### Hi  @edubew 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**